ev3dev2/led.py

```diff
@@ -346,3 +354,214 @@ def all_off(self):
 
         for led in self.leds.values():
             led.brightness = 0
+
+    def reset(self):
+        """
+        Put the LEDs back to their default state of GREEN
+        """
+
+        if not self.leds:
+            return
+
+        self.set_color('LEFT', 'GREEN')
+        self.set_color('RIGHT', 'GREEN')
+
+
+class LedAnimate(threading.Thread):
+
+    def __init__(self):
+        super(LedAnimate, self).__init__()
+        self.leds = Leds()
+
+        # Stop event for animate thread
+        self.done = threading.Event()
+
+    def join(self, timeout=None):
```

Comment:
How about ``stop()`` instead of ``join()``?  I'll add a ``run_for_ms`` option so the user can set a limit on how long it should run.  blocking....need to think about this some, the fact that we are using threads means we aren't going to block.  I could add a ``block`` method to ``LedAnimate`` that drops into a while loop until ``done`` is set.  That feels a little clunky but would work.